contracts/RegisterDomains.sol

```diff
@@ -0,0 +1,55 @@
+//SPDX-License-Identifier: UNLICENSED
```

Comment:
Can this contract really answer most probable questions in terms of data monitoring? -3

Reply:
added: emitting events for registering/releasing events